.github/ISSUE_TEMPLATE/BUG_REPORT.yaml

```diff
@@ -0,0 +1,43 @@
+name: 🐛 Bug report
+description: Create a report to help us improve 🤔.
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for reporting!  Please also use the search to see if there are any other relevant issues or pull requests.
+
+  - type: input
```

Comment:
As this repo is related with a custom theme, I think it would be more helpful to collect info related with the browser rather than with a version